scilpy/viz/backends/fury.py

```diff
@@ -0,0 +1,440 @@
+    scene : window.Scene()
+        Object from Fury containing the 3D scene.
+    """
+
+    scene = window.Scene()
+    scene.background(bg_color)
+
+    # Add actors to the scene
+    for _actor in actors:
+        scene.add(_actor)
+
+    set_viewport(scene, orientation, slice_index, volume_shape, aspect_ratio)
+
+    return scene
+
+
+def create_interactive_window(scene, window_size, interactor,
+                              title="Viewer", open_window=True):
+    """
+    Create a 3D window with the content of scene, equiped with an interactor.
+
+    Parameters
+    ----------
+    scene : window.Scene()
+        Object from Fury containing the 3D scene.
+    window_size : tuple (width, height)
+        The dimensions for the vtk window.
+    interactor : str
+        Specify interactor mode for vtk window. Choices are image or trackball.
+    title : str
+        Title of the scene. Defaults to Viewer.
+    open_window : bool
+        When true, initializes the interactor and opens the window
+        (This suspends the current thread).
+
+    Returns
+    -------
+    show_manager : window.ShowManager()
+        Object from Fury containing the 3D scene interactor.
+    """
+
+    showm = window.ShowManager(scene, title=title,
+                               size=window_size,
+                               reset_camera=False,
+                               interactor_style=interactor)
+    showm.initialize()
+
+    if open_window:
+        showm.start()
+
+    return showm
+
+
+def snapshot_slices(actors, slice_ids, orientation, shape, size):
+    """
+    Snapshot a series of slice_ids from a scene on a given axis_name.
+
+    Parameters
+    ----------
+    actors : list of vtkActor
+        List of actors to snapshot.
+    slice_ids : list of int
+        List of slice indices to snapshot.
+    orientation : str
+        Name of the axis to snapshot.
+    shape : tuple
+        Shape of the volume.
+    size : tuple
+        Size of the viewport.
+
+    Returns
+    -------
+    snapshots : generator of 2d np.ndarray
+        Generator of snapshots.
+    """
+
+    scene = create_scene(actors, orientation, 0, shape, size[0] / size[1])
+
+    for idx in slice_ids:
+        for _actor in actors:
+            set_display_extent(_actor, orientation, shape, idx)
+
+        set_viewport(scene, orientation, idx, shape, size[0] / size[1])
+        yield window.snapshot(scene, size=size).astype(np.uint8)
+
+
+def snapshot_scenes(scenes, window_size):
+    """
+    Snapshot a list of scenes inside a window of given size.
+
+    Parameters
+    ----------
+    scenes : list of window.Scene
+        List of scenes to snapshot.
+    window_size : tuple
+        Size of the window.
+
+    Returns
+    -------
+    snapshots : generator of 2d np.ndarray
+        Generator of snapshots.
+    """
+
+    for scene in scenes:
+        yield window.snapshot(scene, size=window_size)
+
+
+def create_contours_actor(contours, opacity=1., linewidth=3.,
```

Comment:
I don't understand why this is in `fury` backend and not `vtk` backend?

Reply:
Well, the first line of code in the function is using `fury`. And I made sure all rendering stuff is done through `fury` as well, only primitives building and utility with `VTK`. So, all `actor` creation would live here.

Reply:
ok I did not know the first line was FURY.